sources/packages/web/src/constants/routes/RouteConstants.ts

```diff
@@ -0,0 +1,17 @@
+export const routeConstants = {
+  //Student
+  APPSTUDENT: Symbol(),
+  STUDENTDASHBOARD: Symbol(),
+  LOGIN: Symbol(),
+  STUDENTPROFILE: Symbol(),
+  STUDENTPROFILEEDIT: Symbol(),
+  FINANCIALAIDAPPLICATION: Symbol(),
```

Comment:
Please consider using "_" to provide an easier reading.

Reply:
Done